trainer.py

```diff
@@ -30,28 +31,30 @@ def cli_main(
     save_config_callback=None,
     seed_everything_default=42,
     trainer_defaults: dict = {
-        "accelerator": "auto",
+        "accelerator": "gpu",
```

Comment:
Prefer to set `accelerator="auto"` to allow this `trainer.py` to work on the CPU out of the box too.
```suggestion
        "accelerator": "auto",
```

Reply:
Yes, done!